.npmignore

```diff
@@ -13,22 +13,20 @@
 /_site
 /_layouts
 /_includes
-/.github
+/.*
 
-npm-debug.log
-instance.json
-test-gallery.html
-Gemfile.lock
+_config.yml
+bower.json
+codecov.yml
 gallery.html
+Gemfile.lock
 Gemfile
 nodemon.json
+test-gallery.html
+tsconfig.json
 tslint.json
-bower.json
-_config.yml
-.travis.yml
-.jekyll-metadata
-.vscode
-.editorconfig
+yarn.lock
 vl.sublime-project
 **/*.ts
 !**/*.d.ts
```

Comment:
Right now js files are included in release. 

I think in another PR, I want to move all build files (including vega-lite.js|min.js and all compiled js and d.ts to `build/` while keeping the src files in the release).